setup/providers/private-internet-access.md

```diff
@@ -63,6 +63,10 @@ Once enabled, you will keep the same forwarded port for 60 days as long as you b
 
 [@jawilson](https://github.com/jawilson) developed a plugin to automagically update the forwarded port in Deluge: [**deluge-piaportplugin**](https://github.com/jawilson/deluge-piaportplugin)
 
+### qBittorrent
+
+A tool has been developed by the community which automatically updates the forwarded port in qBittorrent: [**qbittorrent-port-updater**](https://github.com/Noah-Huppert/qbittorrent-port-updater)
+
```

Comment:
Can you move this to the setup/advanced/vpn-port-forwarding.md document? There is already a qbittorrent example on how to update the port (update your master branch, this was merged earlier today), but feel free to add a line referencing your tool below that!